app/views/trainees/lead_schools/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<%= render PageTitle::View.new(title: "pages.search_schools.page_title") %>
```

Comment:
Our `PageTitle` component expects this translation to be nested under `components.page_titles.` so this is showing as translation missing at the moment.